packages/models/src/tenant/tenant.ts

```diff
@@ -118,6 +118,7 @@ export const DeclaredTenantAttribute = z.object({
   id: AttributeId,
   assignmentTimestamp: z.coerce.date(),
   revocationTimestamp: z.coerce.date().optional(),
+  delegationId: DelegationId.optional(),
```

Comment:
I think there is one last point where maybe this needs to be added to a converter: https://vscode.dev/github/pagopa/interop-be-monorepo/blob/develop/packages/compute-agreements-consumer/src/converters.ts#L24-L25